mkt/lookup/views.py

```diff
@@ -447,9 +447,10 @@ def _app_downloads(app):
              'alltime': 0}
 
     # Webapps populate these fields via Monolith.
+    # TODO: Update this to use monolith or link to stats or use 90d numbers?
     stats['last_24_hours'] = 'N/A'
-    stats['last_7_days'] = app.weekly_downloads
-    stats['alltime'] = app.total_downloads
+    stats['last_7_days'] = 'N/A'
```

Comment:
@eviljeff I guess just knowing what would be good info there? Also, I removed the totals from the reviewer tools. If you'd like to have the 90-day popularity count I can replace it with that.